pkg/machines/constants/store-action-types.es6

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// --- Store actions --------------------------------------------
+export const ADD_NOTIFICATION = "ADD_NOTIFICATION";
```

Comment:
I can't say I like this very much.. it is yet another place to change when one adds new functionality (but then again with redux this is already ridiculously heavily split), and it makes things harder to debug (as you see numeric values).

But I don't veto this if @mareklibra agrees - at this point the "machines" code is maintained by your team mostly anyway, so you can pick the code style.

Reply:
@martinpitt I don't get your point by "and it makes things harder to debug (as you see numeric values)." Where you 'll see the numeric values?
I don't really see anything against this split. Benefits can be seen here. https://redux.js.org/recipes/reducing-boilerplate#actions
In general I would like to keep somehow naming consistent, and having all action type in one file makes it easier to review.
But I will not insist.

Reply:
> Where you 'll see the numeric values?

I didn't actually see them, but won't they appear in stack traces and similar views, instead of the string constants?

But as I said, I don't veto this for the machines page, it was just a comment.